python-sdk/src/astro/databases/base.py

```diff
@@ -724,8 +746,8 @@ def schema_exists(self, schema: str) -> bool:
     # ---------------------------------------------------------
 
     def get_sqlalchemy_template_table_identifier_and_parameter(
-        self, table: BaseTable, jinja_table_identifier: str
-    ) -> tuple[str, str]:  # skipcq PYL-W0613
+        self, table: BaseTable, jinja_table_identifier: str  # skipcq PYL-W0613
```

Comment:
Why are we bypassing code quality here?

Reply:
`jinja_table_identifier` param is unused here so the static check is complaining about it. Also, this skipcq was there but looks like it was not on the correct line number.